cloud/cloud_env.cc

```diff
@@ -71,7 +71,7 @@ void BucketOptions::TEST_Initialize(const std::string& bucket,
   if (!CloudEnvOptions::GetNameFromEnvironment("ROCKSDB_CLOUD_TEST_BUCKET_NAME",
                                                "ROCKSDB_CLOUD_BUCKET_NAME",
                                                &bucket_)) {
-#ifdef _MSC_VER
+#ifdef _WIN32_WINNT
```

Comment:
This seems to be a different define than most of RocksDB proper uses.  Should this use OS_WIN instead?

Reply:
RocksDB uses _MSC_VER in multiple places. We can use that if you want, but it's less proper (if I'm being pedantic).